README.md

```diff
@@ -12,18 +25,37 @@ For the Points developer take-home assignment, the following task is required: y
     * Displays the amount of taxes owed per band
     * Displays the effective rate
 
-There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+### Level 1
+
+* Fetch the tax rates from this server at: [localhost:5000/tax-calculator/brackets](http://localhost:5000/tax-calculator/brackets)
+
+### Level 2
+
+* Fetch the tax rates by year: [localhost:5000/tax-calculator/brackets/<2019,2020...>](http://localhost:5000/tax-calculator/brackets/2020)
 
+_Warning: this endpoint has gotchas_
+
+## Assignment 2: Autonomous Car
+
+### Instructions
+
+* Download and run this server (see [Getting Started](#getting-started))
+* Build an application / tool in React or Python with the following criteria:
+    * Fetches an autonomous car route from this API server
+    * Determines if the route was a success or a failure
+    * If the route results in a failure, determine the cause as well as the step and position where the failure occurred
```

Comment:
The instructions need more detail. There's no indication that there's only three lanes. Can you update the instructions from here: https://github.com/Points/developer-assignment-instructions/tree/master/autonomous-car